modules/network-observability-dns-overview.adoc

```diff
@@ -19,4 +19,4 @@ When DNS tracking is enabled, you can see the following metrics represented in a
 * Top 5 DNS response code
```

Comment:
think this was missed previosuly but this needs to be updated to `Top X` for all the panel names